client/src/modules/stock/aggregated_consumption/modals/consumption_lot.modal.js

```diff
@@ -0,0 +1,206 @@
+angular.module('bhima.controllers')
+  .controller('StockConsumptionLotsModalController', StockConsumptionLotsModalController);
+
+StockConsumptionLotsModalController.$inject = [
+  'appcache', '$uibModalInstance', 'uiGridConstants', 'data',
+  'SessionService', 'bhConstants', 'AggregateConsumptionModalForm', 'focus',
+];
+
+function StockConsumptionLotsModalController(
+  AppCache, Instance, uiGridConstants, Data, Session, bhConstants, AggregateForm, Focus,
+) {
+  const vm = this;
+
+  const cache = new AppCache('StockEntryModal');
+
+  // initialize the form instance
+  vm.form = new AggregateForm({
+    max_quantity_consumed : Data.stockLine.quantity_consumed,
+    max_quantity_lost : Data.stockLine.quantity_lost,
+    unit_cost : Data.stockLine.unit_cost,
+    lot : Data.stockLine.label,
+    end_date : Data.stockLine.end_date,
+    start_date : Data.stockLine.start_date,
+    rows : Data.stockLine.detailled,
+  });
+
+  vm.hasMissingLotIdentifier = false;
+  vm.hasInvalidLotExpiration = false;
+  vm.hasInvalidLotQuantity = false;
+
+  vm.enterprise = Session.enterprise;
+  vm.stockLine = angular.copy(Data.stockLine);
+
+  vm.entryType = Data.entry_type;
+  vm.isTransfer = (vm.entryType === 'transfer_reception');
+
+  // exposing method to the view
+  vm.submit = submit;
+  vm.cancel = cancel;
+
+  vm.onLotBlur = onLotBlur;
+  vm.onChanges = onChanges;
+  vm.onChangeQuantity = onChangeQuantity;
+  vm.onChangeUnitCost = onChangeUnitCost;
+  vm.onDateChange = onDateChange;
+
+  vm.isCostEditable = (vm.entryType !== 'transfer_reception');
+
+  const cols = [{
+    field : 'status',
+    width : 25,
+    displayName : '',
+    cellTemplate : 'modules/stock/entry/modals/templates/lot.status.tmpl.html',
+  }, {
+    field : 'movement_date',
+    type : 'date',
+    cellFilter : `date:"${bhConstants.dates.format}"`,
+    width : 200,
+    displayName : 'STOCK.AGGREGATED_STOCK_CONSUMPTION.CONSUMPTION_TO_DATE',
+    headerCellFilter : 'translate',
+    cellTemplate : 'modules/stock/aggregated_consumption/templates/lot_movement_date.tmpl.html',
+  }, {
+    field : 'quantity_consumed',
+    type : 'number',
+    width : 150,
+    displayName : 'STOCK.QUANTITY_CONSUMED',
+    headerCellFilter : 'translate',
+    aggregationType : uiGridConstants.aggregationTypes.sum,
+    aggregationHideLabel : true,
+    footerCellClass : 'text-right',
+    cellTemplate : 'modules/stock/aggregated_consumption/templates/consumed.tmpl.html',
+  }, {
+    field : 'quantity_lost',
+    type : 'number',
+    width : 150,
+    displayName : 'STOCK.QUANTITY_LOST',
+    headerCellFilter : 'translate',
+    aggregationType : uiGridConstants.aggregationTypes.sum,
+    aggregationHideLabel : true,
+    footerCellClass : 'text-right',
+    cellTemplate : 'modules/stock/aggregated_consumption/templates/lost.tmpl.html',
+  }, {
+    field : 'actions',
```

Comment:
It would be good to add `width: "10"` here, to limit the width of this field:

![image](https://user-images.githubusercontent.com/896472/109132837-8f4bcd80-7754-11eb-825c-a773675aa511.png)